documentation/src/components/customScope.ts

```diff
@@ -0,0 +1,11 @@
+import { Water } from 'three/examples/jsm/objects/Water';
+import { resetViewerEventHandlers } from "../viewerUtilities";
+import * as THREE from 'three';
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+export const customScope = {
+  resetViewerEventHandlers,
+  THREE,
+  Water,
+  skyUrl: useBaseUrl('/img/sky007.jpg')
+}
```

Comment:
Don't think we need to do it now, but could we in theory make this a function and have another function `addToScope` that allowed us to extend the scope that was provided to the examples?

Reply:
I don't know really. Right now the easiest alternative option I see is having some component that will pollute global scope with custom scope. Something like that:

```mdx
<MyScope scope={{ foo: 'bar' }} />

\```jsx runnable
useFoo(foo);
\```
```

And `MyScope` will basically do something dirty like `Object.assign(window.scope, props.scope)`. Not sure if it's any better than it's now.

Reply:
Works for me, but don't need to do it now